.rubocop_rspec_styleguide.yml

```diff
@@ -10,12 +10,12 @@ RSpec:
 FactoryBot:
   Enabled: false
 
+# Enabled rules
+
+Capybara/NegationMatcher:
+  Enabled: true
+  EnforcedStyle: not_to
```

Comment:
:scream: Oh no, why didn't you choose `to_not`?

It doesn't really matter, but now you opened the flame war. I personally prefer `to_not` because my eye can read this quicker, there's more symmetry:

```rb
expect(page).to have_content "one"
expect(page).to_not have_content "two"

# or
expect(page).to     have_content "one"
expect(page).to_not have_content "two"
```

You are suggesting:
```rb
expect(page).to have_content "one"
expect(page).not_to have_content "two"

# or not aligning as much
expect(page).to     have_content "one"
expect(page).not_to have_content "two"
```

If we look at current use:

```
$ gg -w not_to | wc -l
569
gg -w to_not | wc -l
702
```

It's not a big difference but the current majority it also `to_not`.

Sorry to bring this up. Let's discuss it now and then never bring it up again. :wink: 

What were your reasons `not_to` use `to_not`?

Reply:
Oh, I think I just answered my own question by looking at the docs. The cop knows only one version...

Reply:
😀 
Sorry, the only reason I chose `not_to` is because it was the only option. It's quite odd that `to_not` is not given as an option, maybe we could contribute?
https://docs.rubocop.org/rubocop-capybara/cops_capybara.html#enforcedstyle-not_to

I did briefly consider the style like you've mentioned, thanks for illustrating it further.
I do appreciate the symmetry, and considering that would vote your way. 
I wondered if it could potentially be misread in a large block of code. Could you scan quickly and only see the `to`, missing that one of them is a `to_not`? The best example I could find just now:
```ruby
      expect(job.send(:proxy_orders)).to include proxy_order
      proxy_order.update!(canceled_at: 5.minutes.ago)
      expect(job.send(:proxy_orders)).to_not include proxy_order
      proxy_order.update!(canceled_at: nil)
      expect(job.send(:proxy_orders)).to include proxy_order
      proxy_order.update!(placed_at: 5.minutes.ago)
      expect(job.send(:proxy_orders)).to_not include proxy_order
```
Hmm maybe a more important point is to [not have too many expectations](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecmultipleexpectations)...

So yeah, I like your suggestion, feel free to contribute it ;)

As an aside, I just accidentally discovered someone is building a styleguide for rubocop-capybara, which may be helpful (https://github.com/rubocop/capybara-style-guide/). Looks like it's still a work in progress though.

Reply:
PS I just tried `EnforcedStyle: to_not` just in case it works out of the box... it doesn't :(

```
Error: invalid EnforcedStyle 'to_not' for Capybara/NegationMatcher found in .rubocop_rspec_styleguide.yml
Valid choices are: have_no, not_to
```

Reply:
I just found this one: https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/NotToNot

Reply:
What do you think? https://github.com/openfoodfoundation/openfoodnetwork/pull/12250